src/agoradatatools/great_expectations/gx/json_schemas/genes_biodomains/gene_biodomains_schema.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Is this the generated schema, just included for historic/informational purposes?

Reply:
This schema became part of the expectation suite for `genes_biodomains`. We should keep it in the repository in addition to the expectation suite in case we want to make any future changes. If we make changes to the schema and then run the Jupyter Notebook, those changes will be reflected in the expectation suite.